### PR TITLE
Adds Exa API Key

### PR DESCRIPTION
**Pull request by jwd-dev**
## Describe your changes
Adds Exa as an integration https://docs.exa.ai/reference/search
Adds Jotform as an integration

### CODE REVIEW

File: docs-v2/integrations/all/exa.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Getting started
+
+-   [Generate an API token in your Exa account](https://dashboard.exa.ai/)
+-   [Exa API docs](https://docs.exa.ai/reference/getting-started)
+
+<Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
+
+## API gotchas
+
+<Note>Add Getting Started links and Gotchas by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/peopledatalabs.mdx)</Note>
```
Comment (khaliqgant) [inline docs-v2/integrations/all/exa.mdx]: This link should self reference

**Review by TBonnin — COMMENTED** [review]
thank you for your contribution @jwd-dev 
A link to this new integration should be added to the [`mint.json`](https://github.com/NangoHQ/nango/blob/master/docs-v2/mint.json#L344) file and also probably to https://github.com/NangoHQ/nango/blob/master/docs-v2/integrations/search.mdx

**Comment by jwd-dev** [comment]
Great, I have another one to add so I will do that within this PR. (jotform)

**Comment by jwd-dev** [comment]
It would really be great if it was faster to add generic API key integrations.

**Comment by jwd-dev** [comment]
Within the dev portal or something

**Comment by jwd-dev** [comment]
Added those changes and jotform to the pr @TBonnin

**Pull request merged**